src/rpc/net.cpp

```diff
@@ -1082,7 +1075,7 @@ void RegisterNetRPCCommands(CRPCTable& t)
         {"hidden", &addconnection},
         {"hidden", &addpeeraddress},
         {"hidden", &sendmsgtopeer},
-        {"hidden", &getaddrmaninfo},
+        {"network", &getaddrmaninfo},
```

Comment:
looks like this is ordered by commands using `network` then commands using `hidden` if you could move this up to line 1075

Reply:
done.